### DISCUSSION TITLE
undefined 에러

### DISCUSSION THREAD

**Discussion by jun5640** [naverz/zepeto-studio-kor#344] [5. Q&A]
![image](https://user-images.githubusercontent.com/13131384/153106701-c40c02ea-65e6-497b-93dc-8edb9e5adc19.png)
![image](https://user-images.githubusercontent.com/13131384/153106748-18cdddf4-3d75-4ab7-ac04-1d09ebd784fe.png)

테스트 중에 위와 같은 에러가 납니다. 현재 스키마 타입은 위의 이미지 와 같습니다.

혹시 뭐 때문에 나는 에러인지 알 수 있을까요?

**Comment by runasy-koonta** [comment]
일부분이라도 코드를 첨부해주시면 좋을 것 같습니다!
일단 오류 내용으로 봐서는, this.state.players.get()에서 얻으신 player가 존재하는지 체크해주시면 좋을 것 같아요.
